src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts

```diff
@@ -624,6 +609,37 @@ export class FormPersistentDelegationRequestTransactionTs extends FormTransactio
         );
     }
 
+    private signAndAnnounceTransactions(
```

Comment:
it will be good to add a description for this method.

Reply:
Added.